packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -169,6 +170,14 @@ abstract class FlutterDriver {
   /// Getter of serviceClient.
   VMServiceClient get serviceClient => throw UnimplementedError();
 
+  /// Getter of webDriver.
+  async_io.WebDriver get webDriver => throw UnimplementedError();
+
+  /// Enables accessibility feature.
+  Future<void> enableAccessibility() async {
+    throw UnimplementedError();
+  }
```

Comment:
Shall we add a message to the error?

Reply:
Discussed offline. Will follow existing pattern and do not add message to UnimplementedError.